src/uu/df/src/columns.rs

```diff
@@ -183,4 +183,31 @@ impl Column {
             _ => Err(()),
         }
     }
+
+    /// Return the alignment of the specified column.
+    pub(crate) fn align(column: &Self) -> Alignment {
```

Comment:
A nit, but `align` is a verb and hence this sounds like it would change the alignment somehow, so I'd call this `alignment` instead.

Reply:
Makes sense.